### ISSUE TITLE
Thumbnail metatag not working in Synapse

### ISSUE THREAD

**Issue by joepie91** [revspace/photo-gallery#6]
Synapse isn't detecting the `og:image` (and similar) metatags for some reason. The other metatags are processed correctly. Possibly something to do with the uppercase `.JPG` extension?

**Comment by Peetz0r** [comment]
Nope, we now think it may be because the `content-type` is missing.

It works when I proxy trough Caddy, which somehow automagically adds a `content-type` header.

![image](https://matrix.glitch.im/_matrix/media/r0/download/glitch.im/quwSyZsZvXXnUXyatxlHSGBQ)

See also https://matrix.to/#/!HguitypRcaLhKzXJbP:pixie.town/$9ry7nsNAregHw-MVKBGC84wFxT_g9rpfECpq8wGpO5Y?via=pixie.town&via=matrix.org&via=glitch.im

Credits to @lieuwex
